x-pack/plugins/maps/public/components/map/feature_tooltip.js

```diff
@@ -11,33 +11,71 @@ import { i18n } from '@kbn/i18n';
 
 export class FeatureTooltip extends React.Component {
 
+  _renderFilterButton(tooltipProperty, hasFilters) {
+    if (!this.props.isReadOnly || !hasFilters)  {
```

Comment:
overriding `isReadOnly` to account for the fact that the Maps application does not have a filter bar while dashboard does have a filter bar is brittle. This will break when feature controls are introduced and the Maps application can have isReadOnly set to true. I think we need to either add filter bar to the Maps application as a prerequisite for this PR or add a new boolean in UI state to track whether the container supports filters or not.

Reply:
+1 on introducing new flag isFilterBarPresent